DataConnectors/AWS-S3/ConfigAwsConnector.ps1

```diff
@@ -75,6 +75,7 @@ switch ($AwsLogType)
     "VPC" {.\ConfigVpcFlowDataConnector.ps1; break}
     "CloudTrail" {.\ConfigCloudTrailDataConnector.ps1 ; break }
     "GuardDuty" {.\ConfigGuardDutyDataConnector.ps1 ; break }
+	"CustomLog" {.\ConfigCustomLogDataConnector.ps1 ; break }
```

Comment:
Do you need to update `AwsRequiredPolicies.md` with custom info?
please note that Yechiel wrote this doc to review the text

Reply:
I didn't understand what you mean

Reply:
No, everything is described under Common policies